pages/common/mpc.md

```diff
@@ -0,0 +1,28 @@
+# mpc
+
+> Music Player Client.
+> Program for controlling Music Player Daemon (MPD).
+
+- Toggle between play and pause:
+
+`mpc toggle`
+
+- Stop playing:
+
+`mpc stop`
+
+- Show information about currently playing song:
+
+`mpc status`
+
+- Play next song:
+
+`mpc next`
+
+- Play previous song:
+
+`mpc prev`
+
+- Seek playing song:
```

Comment:
I'm not a native english speaker and wasn't sure using *forward* and *backward* would be correct, mainly because I was thinking *"rewind"* :) , definitely using that instead.